src/main/app/forms/validation/validationConstraints.ts

```diff
@@ -0,0 +1,3 @@
+export class ValidationConstraints {
```

Comment:
this should live next to validationErrors similar responsibility

Reply:
Moved `ValidationErrors`.